index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML / CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+  <article class="product1"> 
+    <header>
+      <h1> Charizard </h1>
+    </header>
+      <section>
+        <img src="http://orig11.deviantart.net/e4ba/f/2013/227/b/b/daveo_s_base_charizard_replica_by_iamthedaveo-d6i7nht.png">
+        <div>
+            <p></p>
+            <p>Charmander Lorem ipsum dolor sit amet, consectetur adipiscing elit. Charmeleon Lorem ipsum dolor sit amet, consectetur adipiscing elit. Charizard Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>  
+        <div>
+          <label>Available</label>
+        </div>
+        <p></p>
+        <p></p>
+      </section>
+      <section>
+        <div>
+          <header>Specifications</header>
```

Comment:
Try to avoid creating divs like this. A `section` is essentially a div but with some semantic meaning of a "section div". Try to just use: `<section><header>...`